src/server_manager/web_app/app.ts

```diff
@@ -140,22 +154,31 @@ type DigitalOceanServerRepositoryFactory = (session: digitalocean_api.DigitalOce
 export class App {
   private digitalOceanRepository: server.ManagedServerRepository;
   private selectedServer: server.Server;
-  private serverBeingCreated: server.ManagedServer;
+  private idServer = new Map<string, server.Server>();
```

Comment:
Can we name this something similar to the `localId` function so that it's clear that they're related?

How about `localServerId(...)` and `localServers`?

Reply:
Renamed `localId` to `localServerId`.
I like naming maps with <key><value> names (<id><Server>). `localServers` sounds like a list or set.
Would 'idServerMap` sound better? My requirement is to clarify what the key and values are, and not just sound like a set.